core-bundle/contao/themes/flexible/styles/main.css

```
@@ -38,6 +38,11 @@ html[data-color-scheme="dark"] {
 	--icon-invisible: url("icons/invisible--dark.svg");
 }
 
+/* Account for the jump links bar */
+html {
+	scroll-padding-top: 36px;
```

```suggestion
	scroll-padding-top: 36px;
	scroll-behavior: smooth;
```

This would improve the usability I think as it shows the user more clearly that the page was scrolled.